openid-connect-4-credential-issuance-1_0.md

```diff
@@ -30,25 +30,25 @@ organization="Microsoft"
 
 .# Abstract
 
-This specification defines an extension of OpenID Connect to allow holders to request issuance and issuers of verifiable credentials issuance of verifiable credentials.
+This specification defines an extension of OpenID Connect to allow holders to request issuance of verifiable credentials.
 
 {mainmatter}
 
 # Introduction
 
-This specification extends OpenID Connect with support for issuance of verifiable credentials, e.g. in the form of W3C Verifiable Credentials. This allows existing OpenID Connect OPs to extends their service and become credential issuer. It also allows new applications built using Verifiable Credentials to utilize OpenID Connect as integration and interoperability layer between credential holders and issuers. 
+This specification extends OpenID Connect with support for issuance of verifiable credentials, e.g. in the form of W3C Verifiable Credentials. This allows existing OpenID Connect OPs to extends their service and become credential issuers. It also allows new applications built using Verifiable Credentials to utilize OpenID Connect as integration and interoperability layer between credential holders and issuers. 
 
-OpenID Connect is an obvious choice for this use case since it already allows Relying Parties to request identity assertions. Verifiable Credentials are very similar in that they allow an Issuer to assert End-User claims. In contrast to identity assertions, a verifiable credential follows a pre-defined schema (the credential type) and is bound to key material allowing the holder to proof the legitimate possession of the credential. This allows presentation of the credential without direct involvment of the credential issuer. This specification caters for those differences. 
+OpenID Connect is an obvious choice for this use case since it already allows Relying Parties to request identity assertions. Verifiable Credentials are very similar in that they allow an Issuer to assert End-User claims. In contrast to the identity assertions, a verifiable credential follows a pre-defined schema (the credential type) and is bound to a key material allowing the holder to prove the legitimate possession of the credential. This allows direct presentation of the credential without involvment of the credential issuer. This specification caters for those differences. 
 
 # Terminology
 
 Credential
 
-A set of one or more claims made by an issuer. (see [@VC_DATA])
+A set of one or more claims made by an issuer. (see [@VC_DATA]) Note that this definition differs from that in [OpenID].
```

Comment:
how does the definition differ and why does it matter? We didn't point that out in OIDC4VPs

Reply:
Differs in a way that those familiar with OIDF world think of credentials as something that gives access to the user claims such as passwords/biometrics as opposed to the claims itself.
probably we should point out in OIDC4VP too. Unless we want it to be a silent evolution of terminology.